tests/unit/test_cost.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime, timedelta
+
+import pytest
+from githubkit.webhooks import WorkflowJob
+
+from gh_actions_exporter.config import Settings
+from gh_actions_exporter.cost import Cost
+
+
+@pytest.fixture
+def headers():
+    headers = {"X-Github-Event": "workflow_run"}
+    return headers
+
+
+def test_get_job_cost():
+    settings = Settings()
+
+    job_request = WorkflowJob(
+        id=123,
+        run_id=456789,
+        run_attempt=1,
+        run_url="https://github.com/user/repo/actions/runs/123",
+        head_sha="a1b2c3d4e5f6g7h8i9j0",
+        node_id="MDg6Q29tbWl0MTIzNDU2Nzg5OnNhbXBsZQ==",
+        name="Job 1",
+        check_run_url="https://api.github.com/repos/user/repo/check-runs/123",
+        html_url="https://github.com/user/repo/actions/runs/123",
+        url="https://api.github.com/repos/user/repo/actions/jobs/123",
+        status="queued",
+        steps=[],
+        conclusion=None,
+        labels=["label1", "label2"],
+        runner_id=None,
+        runner_name=None,
+        runner_group_id=None,
+        runner_group_name=None,
+        started_at=datetime(2021, 1, 1, 0, 0),
+        completed_at=datetime(2021, 1, 1, 0, 0) + timedelta(minutes=10),
+        workflow_name=None,
+        head_branch=None,
+        created_at=datetime.now(),
+    )
```

Comment:
Both workflow_job and settings should be a pytest fixture, two options:

* Option 1: Setup a `tests/unit/conftest.py` with this both this var as fixture
* Option 2: Take the existing workflow_job that is currently set as a dict, reconfigure it as a pydantic object as you did here, and create a `tests/conftest.py` so that both tests can call this feature.

Option 2 is better as there will be less duplicated code but might require a bit more work due to modification of existing tests. Choice is yours.